app/Platform/Enums/RoleName.php

```diff
@@ -6,13 +6,15 @@ enum RoleName: string
 {
     case ADMIN = 'admin';
     case MEMBER = 'member';
+    case CONTENT = 'content';
```

Comment:
content_manager, also rename all references, classes methods and properties accordingly.